### ISSUE TITLE
Updated to 10.13.4 but can't inject intel graphics

### ISSUE THREAD

**Issue by mathguru** [syscl/M3800#88]
Updated to 10.13.4 but can't inject Intel graphics card
here

`<key>Graphics</key>`
`<dict>`
`	<key>Inject</key>`
`		 <dict>`
`			<key>Intel</key>`
**`			<false/>  here I must turn it to false otherwise the startup of OS will be unsuccessful`**
`		</dict>`
`	<key>ig-platform-id</key>`
`	       <string>0x0a2e0008</string>`
`</dict>`

any advice?

**Comment by starfireprime** [comment]
@mathguru - I tried to update as well and was stuck on the Apple logo.  I didn't disable Intel injection to start the OS as your note suggests.  When you disabled injection, were any graphics available?

**Comment by mathguru** [comment]
@starfireprime When I disabled injection, I can login OS, but there is no graphics available.
still working on it. 
someone told me I have to change the type of mac to imac18,1 in the last part of config.plist，but I have't success yet.

**Comment by starfireprime** [comment]
@mathguru - I was able to successfully install and run 10.13.4 in the end by restoring the CLOVER directory that I had for 10.13.3.  The issue was a little confused as I found another thread suggesting that I needed the newest CoreDisplayFixUp and IntelGraphicsFixup.  I placed this in my CLOVER/kext/Other directory before updating.  I believe it had the effect of allowing me to use my laptop display immediately after the first boot and before running syscl's script.  In the end it didn't matter.

1) I installed the 10.13.4 combo update. Rebooted a couple of times and completed the update.
2) First boot into 10.13.4 with the additional kexts in Other allowed me to use my laptop screen without issue.  In the past I would have a display connected to the HDMI port just in case.
3) I ran syscl's script successfully.
4) Upon reboot I could not progress past the "Apple screen" and could not get the graphics to enable.
5) However, upon booting with the EFI/CLOVER on my backup disk and selecting the 10.13.4 partition on the laptop, everything was fine.  So in the end I copied my original 10.13.3 EFI/CLOVER to my boot EFI and I can use 10.13.4 on the laptop.

I noticed in syscl's release notes that he cleaned up some kexts.  After running the script in 10.13.4, there were significantly fewer kexts in the 10.13 directory including no IntelGraphicsFixup, FakeSMC, and Lilo.    I think perhaps it is too clean. :)
